test/CMakeLists.txt

```diff
@@ -36,13 +36,13 @@ macro(onedpl_add_test test_source_file)
 
     add_test(NAME ${_test_name} COMMAND ${CMAKE_CURRENT_BINARY_DIR}/${_test_name})
     if (ONEDPL_DEVICE_TYPE)
-        # TODO: Consider replacement of SYCL_DEVICE_TYPE with a new variable.
-        #       See for details: https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md
-        set(SYCL_DEVICE_TYPE ${ONEDPL_DEVICE_TYPE})
+        string(TOLOWER ${ONEDPL_DEVICE_TYPE} ONEDPL_DEVICE_TYPE)
         if (ONEDPL_DEVICE_TYPE MATCHES "FPGA")
-            set(SYCL_DEVICE_TYPE ACC)
+            set(SYCL_DEVICE_FILTER *:acc)
+        else()
+            set(SYCL_DEVICE_FILTER  *:${ONEDPL_DEVICE_TYPE})
         endif()
-        set_tests_properties(${_test_name} PROPERTIES ENVIRONMENT SYCL_DEVICE_TYPE=${SYCL_DEVICE_TYPE})
+        set_tests_properties(${_test_name} PROPERTIES ENVIRONMENT SYCL_DEVICE_FILTER=${SYCL_DEVICE_FILTER})
```

Comment:
Previously we discussed support of `SYCL_DEVICE_TYPE` for older compiler versions, is it still actual?

Reply:
I think, no. Since we have new compiler in CI, test system, locally. Does anyone see a value to run with old compilers now?